go.mod

```diff
@@ -4,36 +4,36 @@ go 1.20
 
 require (
 	github.com/Masterminds/sprig/v3 v3.2.3
-	github.com/spf13/cast v1.5.0
+	github.com/spf13/cast v1.5.1
 	github.com/stolostron/kubernetes-dependency-watches v0.5.1
-	golang.org/x/exp v0.0.0-20230321023759-10a507213a29
+	golang.org/x/exp v0.0.0-20231006140011-7918f672742d
 	gopkg.in/yaml.v3 v3.0.1
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
-	k8s.io/client-go v0.26.4
+	k8s.io/api v0.28.3
+	k8s.io/apimachinery v0.28.3
+	k8s.io/client-go v1.5.2
```

Comment:
I couldn't find the change log for this. Were you able to see what changed?

Reply:
Kubernetes change logs:
- https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.27.md
- https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.28.md

But for `go-template-utils`, I'm thinking the next version would go into 2.10? It's admittedly lazy, but I'm not sure it's all that worth it at this point to do the juggling to get the release and version bump in the controllers?

Reply:
Then again, I suppose I can just have another commit to make it easier to cherry-pick if needed. I'll do that.